app/src/main/java/io/github/gsantner/memetastic/activity/MainActivity.java

```diff
@@ -177,13 +186,12 @@ protected void onCreate(Bundle savedInstanceState) {
             tab.setText(cat);
             _tabLayout.addTab(tab);
         }
-        _areTabsReady = true;
 
         _viewPager.setAdapter(new MemePagerAdapter(getSupportFragmentManager(), _tagKeys.length, _tagValues));
+        _viewPager.addOnPageChangeListener(this);
```

Comment:
onResume, as removed in onPause

Reply:
Move registering the listener to onResume and unregister it in onPause?

Reply:
y